pkg/controller/verticalpodautoscaler/verticalpodautoscaler_controller.go

```diff
@@ -326,18 +424,39 @@ func (r *Reconciler) AutoscalerDeployment(vpa *autoscalingv1.VerticalPodAutoscal
 	return deployment
 }
 
-// RecommenderPodSpec returns the expected podSpec for the deployment belonging
+// DefaultVPAController returns a default VerticalPodAutoscalerController instance
+func (r *Reconciler) DefaultVPAController() *autoscalingv1.VerticalPodAutoscalerController {
+	var smf float64 = 0.15
```

Comment:
slack discussion... perhaps move these constants to the top and document why they were selected.